components/Badge/__stories__/Badge.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import Badge from '../Badge';
+
+storiesOf('Common/Badge', module)
+  .addDecorator(withKnobs)
+  .add('default', withInfo()(() => <Badge>{text('children', 'PropTypes.string or .node')}</Badge>));
```

Comment:
Can you add a "knob" for each prop?

For `svgComponent`, you can make it a knob type of "select"  and just import a few icons for users to demo between 👍